<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>151</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>身份证号码查询</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(225, 235, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>身份证号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_search">
       <property name="text">
        <string>查  询</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>号    码：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="label_id">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>地    址：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="label_addr">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>生    日：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="label_birth">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>性    别：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLabel" name="label_sex">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>生    肖：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLabel" name="label_animal">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>星    座：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLabel" name="label_star">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
